guacamole/src/main/guacamole-frontend/package.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "frontend",
+    "version": "0.0.0",
+    "scripts": {
+        "ng": "ng",
+        "start": "ng serve",
+        "build": "ng build",
+        "build:app": "ng build guacamole-frontend --base-href /guacamole/",
```

Comment:
I just tried deploying this code and I noticed that all the resources are loaded at `/guacamole/` even though that's not where the webapp is deployed. Is there a way to make this dynamic? We can't safely assume that all users will always deploy their webapps at `/guacamole/`.
![image](https://github.com/apache/guacamole-client/assets/4633119/0fcb79fb-ee6e-46fb-bc01-31fa00196a49)

Reply:
I will add this to the todo list and take a look at it.

Reply:
We internally discussed the issue with the the static base-href configuration. As Angular requires the base path to be set at compile time, dynamically adjusting it isn't straightforward.

Our suggestion would be to keep the `--base-href guacamole` setup for now. However, as a workaround, users who need to deploy the webapp at a different base can manually adjust the base path: they would need to unpack the .war file, edit the base href in the index.html file to the desired path, and then repackage it.